irods/auth/pam.py

```diff
@@ -0,0 +1,9 @@
+class PamLoginException(Exception): pass
+
+def login(conn):
+    if conn.server_version > (4,3):
+        raise PamLoginException('PAM logins in iRODS 4.3+ require a scheme of "pam_password"')
```

Comment:
For my own understanding...

The pam_password scheme continues to use the legacy pam authentication plugin in the server, but is currently using the pam_password scheme name so that users do not need to switch client environments based on whether they are using iCommands or PRC. Is that right?

Reply:
Correct.  The `pam_password` scheme *still* calls the legacy method and uses the legacy APIs, not the new auth framework, within python-irodsclient.  For this to change, I'd need to do the full state machine thing: #499 (issue still open)

Reply:
Excellent, thanks